cmd/operator/kodata/knative-serving/0.17.3/5-serving-post-install-jobs.yaml

```diff
@@ -18,11 +18,11 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  generateName: storage-version-migration-serving-
+  name: storage-version-migration-serving-0.17.3
```

Comment:
Just a nit:

Perhaps, we could just do `storage-version-migration-serving`
Because than in the CSV on the Serverless-Operator, we just change the image, not the name of it ? :man_shrugging:

Reply:
I kept this naming scheme because upstream will give us that for free at HEAD. It replaces generateNames with a $COMPONENT-$VERSION bundle, so we might as well pick that up now.

Reply:
ok - fine w/ that :-)